github_helpers/README.md

```diff
@@ -0,0 +1,32 @@
+# GitHub Helpers
+
+This adds labels to the open PRs based on the location of files changed within the PR.
+
+This will work for admins of SecureDrop that generate a [GitHub token](https://github.com/settings/tokens).
+
+## Setup
+
+### With Docker
+
+    docker build -t github_helpers .
+
+### Without Docker
+
+Download and install `node` and `npm`, then
+
+    npm install
+    cp config.json.example config.json
+
+Enter your GitHub token info into `config.json`.
+
+## Running
+
+### With Docker
+
+Set your `$GITHUB_TOKEN`, and run
```

Comment:
I'm not completely sure I understand your concern, but I've changed the wording to remove the suggestion setting `$GITHUB_TOKEN` in the host env.